site/guide/manage-users.qmd

```diff
@@ -63,17 +72,22 @@ To perform these next steps, you must be a group administrator or a superuser.
 
 4. Click the {{< fa xmark >}} in the pill next to the role for the user that you want to remove.
 
-## Viewing group members
+## User groups
+
+### View group members
```

Comment:
This is also missing a step between 2. and 3. You need to go to **Organization** first.

Reply:
@nrichers - I tested the steps again for this one as well - I do not see the need to go to Org firs

Reply:
Retested, your docs accurately reflect the UI.